uvod-do-programovani-2/slovniky/n-tice.md

```diff
@@ -0,0 +1,38 @@
+## N-tice
+
+Kromě seznamů si povíme o dalších možnostech, jak uložit více hodnot naráz. Datový typ velmi podobný seznamu je <term cs="n-tice" en="tuple">. Představme si ji jako dvojici, trojici, čtveřici apod. nějakých hodnot. Hlavní rozdíly oproti seznamu jsou ty, že není možné měnit prvek na určitém indexu n-tice a není možné žádný prvek přidávat nebo mazat.
+
+N-tici vytvoříme prostou posloupností prvků oddělených čárkou bez hranatých závorek.
+
+```py
+cookies_count = [1, 2, 4, 1, 6, 0, 1]
+print(type(cookies_count))
+
+cookies_count = 1, 2, 4, 1, 6, 0, 1
```

Comment:
Zvážila bych ještě, zda neučit `tuple` jen se závorkami, zejména proto, že je to konzistentnější s výstupem:

```py
t = (1, 2)
print(t)  # (1, 2)
```

@ZelenyMartin Vidíš nějaké výhody zápisu bez závorek?

Reply:
Ty závorky jsou přeci jen navíc. Originální tutorial je taky neukazuje: https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences

Reply:
Jasně, napadá mne, že varianta se závorkami je univerzálnější, protože platí vždy - například v list comprehension je vyžadována: https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions

Za mne je v pohodě to nechat takhle, jen mne zajímalo, jestli to má nějaké další výhody.

Reply:
Mně tam ty závorky přijdou prostě redundantní. Tak počkáme na @pesikj.

Reply:
Já bych byl taky spíše pro závorky. Co jsem viděl použití tuple, tak mi přijde, že většinou to je se závorkami. Výjimka je return u funkce, ale to je spíše takový speciální případ.

Reply:
Shrnutí: Domluveno na slacku, ze to tak muže zůstat.